modules/rosa-sts-creating-cluster.adoc

```diff
@@ -64,11 +64,12 @@ I: To determine when your cluster is Ready, run `rosa describe cluster <cluster_
 $ rosa create cluster --interactive
 ----
 
-. Enter the following command to check the status of your cluster and retrieve your cluster ID. During cluster creation, the `State` field from the output will transition from `pending` to `installing`, and finally to `ready`.
+. Enter the following command to check the status of your cluster and retrieve your cluster ID. During cluster creation,
```

Comment:
Consider this edit: To check the status of your cluster and retrieve your cluster ID, enter the next command. The 'State' field changes from 'pending' to 'installing' to 'ready'.